DB/src/main/java/io/deephaven/db/v2/remote/ConstructSnapshot.java

```diff
@@ -507,24 +507,59 @@ static InitialSnapshot constructInitialSnapshotInPositionSpace(final Object logI
         return snapshot;
     }
 
+    /**
+     * Create a {@link BarrageMessage snapshot} of the specified table using a set of requested columns and positions.
+     * Note that this method uses an index that is in Position space, and that it is notification-oblivious, i.e. it
```

Comment:
This comment and method name don't apply; there's no index, it's the whole table.